src/v2/modules/contacts/validator.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const Joi = require('@hapi/joi');
+
+/**
+ * Last name is always required
+ * One of initials or first name must be supplied
+ */
+const schema = Joi.object({
+  salutation: Joi.string().trim().empty('').optional(),
+  firstName: Joi.string().trim().empty('').optional(),
+  initials: Joi.string().trim().empty('').optional(),
+  lastName: Joi.string().trim().empty('').required(),
+  middleName: Joi.string().trim().empty('').optional(),
+  isTest: Joi.boolean().optional().default(false)
```

Comment:
There are some common validators available in src/v2/lib/validators for fields such as isTest which are shared amongst many entities